exchanges/okex/okex_test.go

```diff
@@ -81,6 +80,103 @@ func areTestAPIKeysSet() bool {
 	return o.ValidateAPICredentials()
 }
 
+func TestUpdateOrderbook(t *testing.T) {
+	tradablePairs, err := o.FetchTradablePairs(asset.Futures)
+	if err != nil {
+		t.Error(err)
+	}
+	if len(tradablePairs) == 0 {
+		t.Error("no tradable pairs")
```

Comment:
t.Fatal, because if this fails, the next lines will cause a panic

Reply:
have updated this in other exchanges too now